client/src/assets/scss/foundation/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$orange: #f17b30;
```

Comment:
These are your custom color variables so they technically don't belong in the `foundation` folder. They can live inside the `scss` folder with `main.scss` and your other custom CSS file friends

Reply:
We moved the _variables.scss one level up.